### PR TITLE
feat: remove unecessary header patch

### PR DESCRIPTION
**Pull request by Sanix-Darker**
## WHAT

- related to : https://github.com/Sanix-Darker/aliash/issues/4

### CODE REVIEW

**Comment by DipandaAser** [comment]
LGTM, for the header patch.

But i have seen another issue, you need to use the return keywords for all the render who comes before the final render.

Ex: in the search handler you'r doing this: 
image

but you must do this:
image

File: backend/handlers.go [hunk 1]
```diff
@@ -120,16 +115,12 @@ func SearchHandler(c *gin.Context) {
...
 
 	c.JSON(http.StatusOK, aliases)
 }
 
 func GetHandler(c *gin.Context) {
-	c.Writer.Header().Set("Content-Type", "application/json")
-
 	uid := c.Param("uid")
 
 	if len(uid) > 0 {
```
Comment (DipandaAser) [inline backend/handlers.go]: error first principle.

handle error before continue.

**Comment by DipandaAser** [comment]
LGTM

**Review by DipandaAser — APPROVED** [review]

**Comment by Sanix-Darker** [comment]
https://github.com/Sanix-Darker/aliash/pull/5/commits/5634fe9f7cadf501b3b0f50dced0a904dc9d9ea3 is so great, to prevent an else block, I had to add it because the code was trying to continue to execute, thanks for the empty return tips 👍🏽

**Pull request merged**